projects/kit/components/checkbox/checkbox-options.ts

```diff
@@ -1,5 +1,19 @@
+import {InjectionToken} from '@angular/core';
 import {TuiSizeL} from '@taiga-ui/core';
 
 export interface CheckboxOptions {
-    size?: TuiSizeL;
+    readonly size: TuiSizeL;
 }
+
+/** Default values for the checkbox options. */
+export const TUI_CHECKBOX_DEFAULT_OPTIONS: CheckboxOptions = {
+    size: 'm',
+};
+
+/** Injection token that can be used to specify checkbox options. */
+export const TUI_CHECKBOX_OPTIONS = new InjectionToken<CheckboxOptions>(
```

Comment:
@fynnfeldpausch @MarsiBarsi so what do you think if we will use this same token for icons config? If we will, then we need to move that to `PrimitiveCheckbox`. Sounds like a reasonable thing to do, I wouldn't want to create too many tokens.

Reply:
Good question. Don't know how to achieve a good separation then, as the checkbox might define other inputs as the primitive checkbox. One could either use different tokens for every component type (checkbox, checkbox-block, primitive checkbox,...) or go for a unified one with all fields and use it in every component. Don't know where to place it then. More interestingly, how would one make a distinction between the default size of the primitive checkbox and the normal checkbox component. Or use one size option and use it in every component (so basically not making them configurable individually)?...

Reply:
I was thinking one token per component. In this particular case, looks like Checkbox will just inject the same token.